### DISCUSSION TITLE
Avoid re-publishing migration if already present

### DISCUSSION THREAD

**Discussion by juvid** [spatie/laravel-medialibrary#2296] [General]
Hi

The same problem exists in this project like spatie/laravel-permission#960

Running php artisan vendor:publish --tag=migrations publishes a new migration file from this package, with a new timestamp every time.

**Comment by drbyte** [comment]
One could argue that this is "by design", as the package's core migration file changes over time. 
If you re-publish the migration then you can compare the files to make sure you're aware of any changes you need to make manually to your prior database (eg: by creating your own migration to accommodate the changes).
